mesonbuild/minstall.py

```diff
@@ -463,8 +463,8 @@ def do_symlink(self, target: str, link: str, full_dst_dir: str) -> bool:
         abs_target = target
         if not os.path.isabs(target):
             abs_target = os.path.join(full_dst_dir, target)
-        if not os.path.exists(abs_target):
-            raise RuntimeError(f'Tried to install symlink to missing file {abs_target}')
+        #if not os.path.exists(abs_target):
+        #    raise RuntimeError(f'Tried to install symlink to missing file {abs_target}')
```

Comment:
This is a TODO: when installing only `devel` the target does not exist, not sure we should always allow installing symlink to a target that does not exist??

Reply:
Allowed it only for library symlinks for now. Better be conservative.